boards/shields/adafruit_winc1500/adafruit_winc1500.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2020 Kim Bøndergaard, <kim.bondergaard@prevas.dk>, Prevas A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	status = "okay";
+	cs-gpios = <&arduino_header 16 0>;			/* D10 */
+
+	winc1500@0 {
+		status = "ok";
+		compatible = "atmel,winc1500";
+		reg = <0x0>;
+		label = "winc1500";
+		spi-max-frequency = <4000000>;
+		irq-gpios = <&arduino_header 13 GPIO_ACTIVE_LOW>;	/* D7 */
+		reset-gpios = <&arduino_header 11 GPIO_ACTIVE_LOW>;	/* D5 */
+		enable-gpios = <&arduino_header 12 0>;			/* D6 */
```

Comment:
`enable-gpios = <&arduino_header 12 GPIO_ACTIVE_HIGH>;` ?

Reply:
Probably also right. Looking into the code base I think using 0 is very common. 
Are there any recommendations

Reply:
GPIO_ACTIVE_HIGH instead of 0 is self explaining.